setup.py

```diff
@@ -5,12 +5,13 @@
 
 setuptools.setup(
     name='caper',
-    version='1.0.0',
+    version='1.0.1',
```

Comment:
Would make sense to do a minor version bump.
```suggestion
    version='1.1.0',
```

Reply:
Thanks, bumped it to 1.1.0.